HousingSearchApi/Startup.cs

```diff
@@ -126,6 +131,7 @@ public static void ConfigureServices(IServiceCollection services)
             RegisterGateways(services);
```

Comment:
You need to call the ConfigureLogging method that has been added. I.e.
```ConfigureLogging(services, Configuration);```

Reply:
I've added this